plenum/server/node.py

```diff
@@ -1649,6 +1649,19 @@ def has_ordered_till_last_prepared_certificate(self) -> bool:
             return True
         return compare_3PC_keys(lst, self.master_replica.last_ordered_3pc) >= 0
 
+    def is_catch_up_limit(self):
```

Comment:
`is_catch_up_limit`'s meaning is hard to guess, change it to `has_catchup_limit_reached`?